cdbcli/environment.py

```diff
@@ -1,20 +1,63 @@
+import contextlib
+import io
+import subprocess
 import sys
 
 
-class Environment(object):
+class Environment():
```

Comment:
OK so judging by [this](http://stackoverflow.com/a/9448136) and [this](http://docs.python.org/py3k/howto/pyporting.html#subclass-object)  Totally valid to do:

```
class Environment:
```

The only issue is that while you get new style classes in Python 3 with this.  This is not the case with Python 2.  So if you want to work with both versions of Python, it would be better to go with:

```
class Environment(object);
```

Again, it all depends on whether or not you *really* want the code to work in both Python 2 and Python 3.